tools/project_manager/show_all_good_word_suggestions.php

```diff
@@ -79,7 +99,9 @@
     // get all projects for this PM
     $projects = _get_projects_for_pm($pm);
 
+    $rejectAlllabel = _("Reject all suggestions");
     $submitLabel = _("Add selected words to Good Words List");
+    $rejectLabel = _("Add selected words to Bad Words List");
```

Comment:
> Am I correct that "Reject all suggestions" is just a means of clearing the flag so that a PM knows they've finished with this word list? That is, the words that were rejected wholesale are not on a list to be used as a filter for future suggestions and never seen again?

Correct.

Reply:
Thanks for the confirmation. Either way, it needs to be documented exactly how it works, so there's no misunderstanding. Possibly as part of the recent additions announcement.